git-sidecar/Dockerfile.arm64v8

```diff
@@ -0,0 +1,13 @@
+FROM arm64v8/alpine:3.8
```

Comment:
Was there a reason to keep this arch-specific Dockerfile around?  Or can it be removed now that they are dynamically generated?

Reply:
Oversight. Will remove it.